packages/clarity-js/test/core.test.ts

```diff
@@ -33,21 +33,23 @@ describe('Core Tests', () => {
         let email = node(decoded, "attributes.id", "eml");
         let password = node(decoded, "attributes.id", "pwd");
         let search = node(decoded, "attributes.id", "search");
+        let card = node(decoded, "attributes.id", "cardnum");
         let click = clicks(decoded)[0];
         let input = inputs(decoded)[0];
         
         // Non-sensitive fields continue to pass through with sensitive bits masked off
-        assert.equal(heading, "Thanks for your order #••••••••");
+        assert.equal(heading, "Thanks for your order #▫▪▪▫▫▫▪▪");
 
         // Sensitive fields, including input fields, are randomized and masked
         assert.equal(address, "•••••• ••••• ••••• ••••• ••••• •••••");
         assert.equal(email.attributes.value, "••••• •••• •••• ••••");
         assert.equal(password.attributes.value, "••••• ••••");
-        assert.equal(search.attributes.value, "hello •••••");
+        assert.equal(search.attributes.value, "hello ▪▫▪▪▪");
```

Comment:
What is the difference between the different characters for masking here (the solid and non-solid squares)?